index.html

```diff
@@ -59,6 +59,87 @@ <h1 id="title">
         This is a work in progress.
       </p>
     </section>
+    <section>
+      <h2>
+        Introduction
+      </h2>
+      <p>
+        HTML allows the display of many media types through elements such as
+        &lt;img&gt;, &lt;picture&gt;, or &lt;video&gt;, but it does not provide
```

Comment:
This should be written in such  way that it describes not what HTML is lacking (i.e., 3D), but that this is _that feature_. Put differently, it should be written as if someone from the future is reading the standardized solution, not as something that is being proposed. 

Does that make sense?